src/openshift/openshiftItem.ts

```diff
@@ -55,6 +55,18 @@ export abstract class OpenShiftItem {
         return (validator.matches(value, '^[a-z]([-a-z0-9]*[a-z0-9])*$')) ? null : message;
     }
 
+    static hideTokenpassword(value: string) {
+        const regex = /--token\s*=\s*(.\w[a-zA-Z0-9!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]*)/;
```

Comment:
This regex looks quite complicated. Not sure if tokens can contain spaces, but if not, it can be simplified to this:
```typescript
/--token\s*=\s*([^\s]*)/
```
Or a more universal regex can be used, like the one I suggested for the password masking.